pkg/kube/snapshot/types.go

```diff
@@ -0,0 +1,35 @@
+package snapshot
+
+// SnapshotMeta is metadata of snapshot which is configurable.
+type SnapshotMeta struct {
```

Comment:
I would prefer `ObjectMeta`, so that on import it becomes `snapshot.ObjectMeta`.